src/main/java/juniorJavaDevClass/Files/Read.java

```diff
@@ -0,0 +1,36 @@
+package juniorJavaDevClass.Files;
+
+import java.io.*;
+
+public class Read {
+    public static void main(String[] args) {
+        // 1. Define the file (path) that we want to read
+        String filename = "C:\\Users\\leilanitaylor\\OneDrive - Clearwater\\Documents\\Java Learning\\Files\\FileToRead.txt";
```

Comment:
I'd be careful about having a file only accessible via your CW account consumed by a program in your personal github account.